app/models/dish.rb

```diff
@@ -1,4 +1,11 @@
 class Dish < ApplicationRecord
   validates_presence_of :name, :description
   belongs_to :chef
+
+  has_many :dish_ingredients, dependent: :destroy
+  has_many :table_ingredients, through: :dish_ingredients
+
+  def total_calories_count
+    table_ingredients.sum(:calories)
```

Comment:
Excellent use of the aggregate `sum` function! Nice work encapsulating this into a model method, too, which follows MVC principles